gradio/components/highlighted_text.py

```diff
@@ -158,8 +158,9 @@ def postprocess(
                 entities = sorted(entities, key=lambda x: x["start"])
                 for entity in entities:
                     list_format.append((text[index : entity["start"]], None))
+                    entity_group = "entity_group" if "entity_group" in entity else "entity"
                     list_format.append(
-                        (text[entity["start"] : entity["end"]], entity["entity"])
+                        (text[entity["start"] : entity["end"]], entity_group)
```

Comment:
This should be 
```suggestion
                        (text[entity["start"] : entity["end"]], entity[entity_group])
```
right?